rocky/rocky/templates/tasks/boefjes.html

```diff
@@ -56,7 +66,7 @@ <h1>{% translate "Boefjes" %}</h1>
                                         </td>
                                     </tr>
                                     <tr class="expando-row">
-                                        <td colspan="5">
+                                        <td colspan="{% if not organization.code %}7{% else %}5{% endif %}">
```

Comment:
The "modified at" column is new and not conditional, right? So this should be 6 instead of 5?
```suggestion
                                        <td colspan="{% if not organization.code %}7{% else %}6{% endif %}">
```